README.markdown

```diff
@@ -362,6 +362,9 @@ Generated code will be placed in the Gradle build directory.
 
 - With `--ts_proto_opt=outputPartialMethods=false`, the `Message.fromPartial` methods for accepting partially-formed objects/object literals will not be output.
 
+- With `--ts_proto_opt=forcePartials=true`, the generated interfaces and clients for services will accept partials.  This is less type-safe, and similar to `optionalTypes=all` without adding ?'s to all of your interfaces.
```

Comment:
Wdyt about `useOptionals=services` as a way to ask for services to use partials but not the messages themselves?